common/djangoapps/entitlements/tests/test_models.py

```diff
@@ -168,13 +168,29 @@ def test_expired_at_datetime(self):
         self.enrollment.created = past_datetime
         self.course.start = past_datetime
         entitlement.save()
+        entitlement.refresh_from_db()
```

Comment:
Is this line needed after a save()?  If so, you probably need it in the block below.

Reply:
Actually I can probably remove this, let me test locally really quickly since I don't use it in other places.